BoxContentSDK/BoxContentSDK/Categories/UIApplication+ExtensionSafeAdditions.m

```diff
@@ -12,18 +12,23 @@ + (UIApplication *)box_sharedApplication
 {
 #ifndef BOX_TARGET_IS_EXTENSION
```

Comment:
synced up offline. This compiles regardless of the target since it's a dylib. need to remove compile time checks for runtime checks.